crawl-ref/source/dat/database/monspeak.txt

```diff
@@ -5042,6 +5083,28 @@ _Maggie_rare_
 @The_monster@ shouts @at_foe@, "Leave me alone!"
 
 @The_monster@ mutters to herself, "Master Lance won't take me back. This is the way forward."
+%%%%
+Maggie killed
+
+w:10
+__NONE
+
+w:1
+@The_monster@ bemoans, "Does the illustrious tale of @The_monster@ really end here?"
+
+@The_monster@ bemoans, "I'm too young to die!"
+
+@The_monster@ pouts, "I had so many more dragons to slay!"
+%%%%
+Maggie banished
+
+w:10
+__NONE
+
+w:1
+@The_monster@ groans, "Eek! Not the Abyss! That place is such a drag!"
+
+@The_monster@ says wistfully, "How many dragons are there in the Abyss, I wonder?"
```

Comment:
and because weight 10 is the default, this line is just as likely as __NONE. Was w:1 your intention for all the spoken lines?

Reply:
My intention was to make the death/banishment lines a lower likelihood than "no message"

Reply:
Yes, w:1 for all of them - somehow I thought that each line below w:1 would have that weight applied. I'll change it.